public/data/job-openings-data.json

```diff
@@ -707,6 +707,49 @@
           "experience": "1+ year"
         }
       ]
+    },
+    {
+      "id": 17,
+      "companyName": "Inferenz Tech Private Limited",
+      "location": "Ahmedabad, India",
+      "bio": "Inferenz breaks the barriers to digitalization by uncomplicating the journey and helping customers discover complete clarity. We are a group of technologists and domain experts that provides new age technologies to adapt to change and accelerate business growth. We are obsessed with staying ahead of the technology curve. Our team of innovative technologists and domain experts are constantly working on improving our technologies to keep delivering the best solutions to our customers.",
+      "companyLogoUrl": "https://drive.google.com/drive/folders/13qhNUhPGCz0FlENmU30NF86qxCBbOijO",
```

Comment:
companyLogoUrl will not work. Pleaser refer other job objects

Reply:
Updated that